client/components/faq/style.scss

```diff
@@ -35,10 +35,12 @@ $faq-gutter-width: 24px;
 	padding-left: ($faq-gutter-width / 2);
 	padding-right: ($faq-gutter-width / 2);
 
-	@media
-		( min-width: 480px ) and ( max-width: 660px ),
-		( min-width: 800px ) and ( max-width: 1040px ) {
-			width: 50%;
+	@include breakpoint( '480px-660px' ) {
```

Comment:
Can we combine this and the other breakpoint below it? Rather than duplicating it might be better to just do something like:

```
@include breakpoint ( '>480px' ) {
    width: 50%;
}
```

And then the `>1040px` breakpoint is picked up below it.

Reply:
Scratch that. I forgot about the sidebar between `660px` and `800px`.